packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -341,9 +345,27 @@ describe('Checkout', () => {
                     localVue,
                     propsData
                 });
+                await flushPromises();
 
                 expect(loadAvailableFulfilmentSpy).toHaveBeenCalled();
             });
+
+            describe('if isLoggedIn set to `false`', () => {
```

Comment:
We should have the opposite case tested as well. If it's set to `true`, we should verify there's no call made.

Reply:
Fixed